src/ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -322,53 +348,47 @@ def __init__(
             print_com=print_com,
             **start_parm,
         )
-        self._mode = "grpc"
+        self._mode: Literal["grpc"] = "grpc"
 
         # gRPC request specific locks as these gRPC request are not thread safe
-        self._vget_lock = False
-        self._get_lock = False
-
-        self._prioritize_thermal = False
-        self._locked = False  # being used within MapdlPool
-        self._stub = None
-        self._cleanup = cleanup_on_exit
-        self.__remove_temp_dir_on_exit = remove_temp_dir_on_exit
-        self._jobname = start_parm.get("jobname", "file")
-        self._path = start_parm.get("run_location", None)
-        self._busy = False  # used to check if running a command on the server
-        self._local = ip in ["127.0.0.1", "127.0.1.1", "localhost"]
+        self._vget_lock: bool = False
+        self._get_lock: bool = False
+
+        self._prioritize_thermal: bool = False
+        self._locked: bool = False  # being used within MapdlPool
+        self._stub: Optional[mapdl_grpc.MapdlServiceStub] = None
+        self._cleanup: bool = cleanup_on_exit
+        self.__remove_temp_dir_on_exit: bool = remove_temp_dir_on_exit
+        self._jobname: str = start_parm.get("jobname", "file")
+        self._path: str = start_parm.get("run_location", None)
+        self._busy: bool = False  # used to check if running a command on the server
+        self._local: bool = ip in ["127.0.0.1", "127.0.1.1", "localhost"]
         if "local" in start_parm:  # pragma: no cover  # allow this to be overridden
-            self._local = start_parm["local"]
-        self._health_response_queue = None
-        self._exiting = False
-        self._exited = None
-        self._mute = False
-        self._db = None
-        self.__server_version = None
-
-        # saving for later use (for example open_gui)
-        start_parm["ip"] = ip
-        start_parm["port"] = port
-        self._start_parm = start_parm
-
-        if port is None:
-            from ansys.mapdl.core.launcher import MAPDL_DEFAULT_PORT
-
-            port = MAPDL_DEFAULT_PORT
-        self._state = None
-        self._stub = None
-        self._timeout = timeout
-        self._pids = []
+            self._local: bool = start_parm["local"]
+        self._health_response_queue: Optional[Queue] = None
```

Comment:
`Queue` is missing among the import
If you happen to know the type of the variables inside the Queue you should specify it
```python
Optional[Queue[TypeOfElementsInside]]
```
Personally I didn't find so if it's also your case you can keep `Optional[Queue]`

Reply:
I have no idea what is the Queue is for. Furthermore, this was implemented a long ago, and I don't think it is used.